src/app/(auth)/sign-in/signin-form.tsx

```
@@ -23,29 +23,34 @@ import { useForm } from "react-hook-form";
+          message:
+            "Username must be all lowercase. Without numbers or special characters.",
+        }),
+      usernameId: z
+        .string()
+        .min(5, {
+          message: "The ID for the username must be 5 characters long.",
+        })
+        .max(5, {
+          message: "The ID for the username must be 5 characters long.",
+        })
+        .refine((val) => !isNaN(Number(val)), {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)